src/blocked.c

```diff
@@ -678,8 +678,7 @@ void handleClientsBlockedOnKeys(void) {
                 unblockDeletedStreamReadgroupClients(rl);
                 /* Edge case: If lookupKeyReadWithFlags decides to expire the key we have to
                  * take care of the propagation here, because afterCommand wasn't called */
-                if (server.also_propagate.numops > 0)
-                    propagatePendingCommands();
+                postUnitOperations();
```

Comment:
some comments before the call, look like they're outdated, i.e. the comment in this one speaks about lazy expiry.
maybe if the call is renamed from "Unit" to "Modification" or "Action" it can still make sense, but if we stay with "Unit" i feel we need to improve some of the comments. (even if just saying "the following call handles that").